src/libs/antares/study/area/area.h

```diff
@@ -279,7 +279,7 @@ class Area final : private Yuni::NonCopyable<Area>
     PartRenewable renewable;
     //@}
 
-    ShortTermStorage::Container shortTermStorage;
+    ShortTermStorage::STstorageInput shortTermStorage;
```

Comment:
Please avoid changing class definitions in the skeleton branch

See [this PR](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1163)